src/traccuracy/matchers/_ctc.py

```diff
@@ -51,8 +51,11 @@ def _compute_mapping(self, gt_graph: TrackingGraph, pred_graph: TrackingGraph):
         if mask_gt.shape != mask_pred.shape:
             raise ValueError("Segmentation shapes must match between gt and pred")
 
-        mapping = []
+        mapping: list[tuple] = []
         # Get overlaps for each frame
+        if gt.start_frame is None or gt.end_frame is None:
```

Comment:
Just checking my understanding, could this also be an `and` because we wouldn't expect the start or end frame to be None without the other also being none?

Reply:
Right, because they are always set together